BraveWallet/Settings/ManageSiteConnectionsView.swift

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveUI
+import SwiftUI
+import struct Shared.Strings
+
+struct ManageSiteConnectionsView: View {
+
+  @ObservedObject var siteConnectionStore: ManageSiteConnectionsStore
+  @State private var filterText: String = ""
+  @State private var isShowingConfirmAlert: Bool = false
+  
+  var body: some View {
+    List {
+      ForEach(siteConnectionStore.siteConnections.filter(by: filterText)) { siteConnection in
+        NavigationLink(
+          destination: SiteConnectionDetailView(
+            siteConnection: siteConnection,
+            siteConnectionStore: siteConnectionStore
+          )
+        ) {
+          SiteRow(
+            siteConnection: siteConnection
+          )
+          .osAvailabilityModifiers { content in
+            if #available(iOS 15.0, *) {
+              content
+                .swipeActions(edge: .trailing) {
+                  Button(role: .destructive, action: {
+                    withAnimation {
+                      siteConnectionStore.removeAllPermissions(from: [siteConnection])
+                    }
+                  }) {
+                    Label(Strings.Wallet.delete, systemImage: "trash")
+                  }
+                }
+            } else {
+              content
```

Comment:
How to remove the site connection on iOS 14 then? is there any other way than swipe action

Reply:
On iOS 14, it will use the `onDelete` closure on the `List` below. On iOS 15+, this swipe actions modifier takes over and is used instead.